### ISSUE TITLE
[Bug]private module could be imported from plain script

### ISSUE THREAD

**Issue by jondy** [dashingsoft/pyarmor#1610]
Reproduced steps:
```
$ echo "print('OK')" > foo.py
$ pyarmor gen --private foo.py
INFO     Python 3.12.0
INFO     Pyarmor 8.4.5 (pro), 005068, btarmor
INFO     Platform darwin.x86_64
INFO     search inputs ...
INFO     find script tsrc/foo.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'darwin.x86_64'}
INFO     write dist/pyarmor_runtime_005068/pyarmor_runtime.so
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "foo"
INFO     obfuscating file foo.py
INFO     write dist/foo.py
INFO     obfuscate scripts OK
$ cd dist
$ echo "import foo" > main.py
$ python main.py
$
```

**Expected results**

It raises RuntimeError `unauthorized use of script`

**Actual results**

No any exception raised.

**Comment by jondy** [comment]
It will be fixed in next `pyarmor.cli.core` release

Verify it by downloading dev version
http://pyarmor.dashingsoft.com/downloads/temp/5.4.3/windows.x86_64/libs/cp3.8/pyarmor_runtime.pyd

And overwrite `/path/to/package/pyarmor/cli/core/pyarmor_runtime.pyd`

**Comment by jondy** [comment]
Fixed in 8.4.6
